atlp-rwanda/Team-Sostene-E-commerce-fe
# #185172119 implement change password

#### What does this PR do?

-ensures a user changes the password to a new one
-ensure the new password is strong as the old one
- make sure the form is validated before making an API call

#### How should this be manually tested?
- login as a user
- on navigation go to change password
- enter the old ,new and confirm the password
- the new password must not be the same as the old password

#### What are the relevant pivotal tracker/Trello stories?

#185172119
